doc/api/worker_threads.md

```diff
@@ -1064,6 +1068,15 @@ added: v10.5.0
 The `'online'` event is emitted when the worker thread has started executing
 JavaScript code.
 
+### Event: `'timeout'`
```

Comment:
Not sure why this is needed? Does it exists in child_process?

Reply:
If one needs to know when a timeout happened, he can use `AbortSignal.timeout` which already includes events